kura/test/org.eclipse.kura.core.net.test/src/test/java/org/eclipse/kura/core/net/WifiAccessPointImplTest.java

```diff
@@ -0,0 +1,497 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Eurotech and/or its affiliates and others
```

Comment:
So it would be then - `Copyright (c) 2016, 2017 Eurotech and/or its affiliates and others`